### ISSUE TITLE
Add "Sponsors" section

### ISSUE THREAD

**Issue by egillespie** [lansingcodes/www#9]
Many of the free tech events in Lansing would not be possible without the generosity of sponsors. Not only that, many sponsors also provide technical, recruiting, and other services that are beneficial to tech enthusiasts.

A "Sponsors" section on Lansing.Codes would call attention to the additional support we receive (Meetup no longer displays sponsors on event pages) and offer event attendees a common place to visit if they are looking for sponsors for other events or interested in other services that sponsors may offer.

The "Sponsors" section should include an image, link, name, and description of each sponsor. A link to a page that explains how to become a sponsor, including an application form, would also be an easy way to discover other businesses interested in sponsoring events. An additional link for event organizers to request a sponsor be added to the page would also be useful for maintaining this section.

Current list of sponsors:

* [TEKsystems](https://www.teksystems.com/)
* [RMC Agency](http://www.rmcagency.com/)
* [SBDC Lansing](https://www.facebook.com/LansingSBDC/)
* [A2 Hosting](https://www.a2hosting.com/)
* [LEAP](http://www.purelansing.com/)

**Comment by dhcrain** [comment]
I am looking at tackling this. I think a similar approach to the meetup section could be a good way to go. thoughts @egillespie .

**Comment by egillespie** [comment]
@dhcrain I agree! A responsive section that shows at least the logo and maybe company name. But don't let me hold you back — do whatever you think will look and work great!

**Comment by dhcrain** [comment]
@egillespie Looks like we still need to add a form to this section, do you have thoughts on that? were you thinking along the lines of a google form or something else?
What info would need to be captured?

**Comment by egillespie** [comment]
Created #49 to address the sponsorship form.
